src/ray/gcs/gcs_client/accessor.cc

```diff
@@ -21,10 +21,23 @@
 #include "ray/gcs/gcs_client/gcs_client.h"
 
 namespace {
-inline int64_t GetGcsTimeoutMs() {
+
+inline int64_t GetDefaultGcsTimeoutMs() {
   return absl::ToInt64Milliseconds(
       absl::Seconds(RayConfig::instance().gcs_server_request_timeout_seconds()));
 }
+
+/// Gets the timeout in milliseconds, used by a GCS Client RPC call.
+/// -1 means infinite retry.
+/// 0 means system default (= RAY_gcs_server_request_timeout_seconds)
```

Comment:
Should we just ask callers to pass in `GetDefaultGcsTimeoutMs()` if they want the default so we don't need to assign special meaning for `0` here.

If you check gcs_rpc_client.h, only -1 has special meaning.

Reply:
done